File: src/Book.java
public class Book {

    private final long id;
    private final long score;

    Book (long id, long score) {
        this.id = id;
        this.score = score;
    }

    public long getId() {
        return id;
    }

    public long getScore() {
        return score;
    }
}

File: src/Python/Book.py
class Book:
    def __init__(self):
        self._bookId = -1
        self._bookScore = -1

    @property
    def bookId(self):
        return self._bookId

    @bookId.setter
    def bookId(self, nId):
        self._bookId = nId

    @property
    def bookScore(self):
        return self._bookScore

    @bookScore.setter
    def bookScore(self, score):
        self._bookScore = score